docs/data-sources/package.md

```diff
@@ -131,14 +131,14 @@ Read-Only:
 
 - `examples` (List of Object) (see [below for nested schema](#nestedobjatt--spec--values_schema--template--raw--examples))
 - `properties` (List of Object) (see [below for nested schema](#nestedobjatt--spec--values_schema--template--raw--properties))
-- `title` (String) Title of object.
+- `title` (String)
```

Comment:
I am guessing these are auto-generated changes, but can you please exclude them from this MR?

Reply:
@asutoshpalai  These were auto generated with tfplugindocs as per @TaylorOno  request and I have generated docs few times across pushes. It will be very difficult to separate from the MR now.